packages/amplify-category-function/src/provider-utils/awscloudformation/service-walkthroughs/execPermissionsWalkthrough.ts

```diff
@@ -183,8 +183,8 @@ export async function askExecRolePermissionsQuestions(context, allDefaultValues,
   }
 
   allDefaultValues.categoryPolicies = categoryPolicies;
-  const resourceProperties = [];
-  const resourcePropertiesJSON = {};
+  let resourceProperties = [];
+  let resourcePropertiesJSON = {};
```

Comment:
put these back to const please, then LGTM.

Reply:
fixed